server/verify_sev.go

```diff
@@ -5,38 +5,54 @@ import (
 	spb "github.com/google/go-sev-guest/proto/sevsnp"
 	"github.com/google/go-sev-guest/validate"
 	sv "github.com/google/go-sev-guest/verify"
-	"github.com/google/go-sev-guest/verify/trust"
 )
 
+// The policy on GCE is to allow SMT, and eventually MigrateMA, but no debug bit.
+var defaultSevSnpGuestPolicy = sabi.SnpPolicy{
+	SMT:       true,
+	MigrateMA: true,
+}
+
 // VerifySnpOpts allows for customizing the functionality of VerifyAttestation's SEV-SNP verification.
 type VerifySnpOpts struct {
-	// ReportData is the expected REPORT_DATA field of the attestation report.
-	ReportData [sabi.ReportDataSize]byte
-	// TrustedRoots defines which AMD root (ARK) and intermediate (ASK) keys to trust
-	// to verify a versioned chip endorsement key (VCEK) that signs attestation reports.
-	// If nil, falls back on go-sev-guest's embedded root certs. Maps a product name
-	// to an array of allowed roots.
-	TrustedRoots map[string][]*trust.AMDRootCerts
-	// Allow the debug bit to be set (should only be used for testing).
-	AllowDebugTestOnly bool
-	// Getter is the object that will fetch files from URLs.
-	Getter trust.HTTPSGetter
+	Validation   *validate.Options
+	Verification *sv.Options
+}
+
+// SevSnpDefaultPolicy returns a default validation policy for SEV-SNP attestation reports on GCE.
+func SevSnpDefaultPolicy(tpmNonce []byte) *validate.Options {
```

Comment:
nit: should this be something like `SevSnpDefaultValidateOpts`?

Reply:
Done.